spacy/cli/templates/quickstart_training.jinja

```diff
@@ -32,9 +32,10 @@ batch_size = {{ 128 if hardware == "gpu" else 1000 }}
 factory = "transformer"
 
 [components.transformer.model]
-@architectures = "spacy-transformers.TransformerModel.v1"
+@architectures = "spacy-transformers.TransformerModel.v2"
 name = "{{ transformer["name"] }}"
 tokenizer_config = {"use_fast": true}
+transformer_config = {"output_attentions": false}
```

Comment:
@danieldk : should we bump to v3 in the quickstart too? And add the default arguments for `mixed_precision` and `grad_scaler_config`? I know the feature is experimental and the thinc side of things might change, but I suppose these two arguments will remain roughly the same?

Reply:
I have updated the `v2` references to `v2`. I am not sure about the quickstart. People are probably still using Pascal-generation GPUs like the 1080Ti for finetuning transformers, which support FP16 but do not have Tensor Cores, and do usually not benefit from mixed-precision training.

Maybe we should add the two options to the example, but suggest `mixed_precision = false` by default?

Reply:
Yea I agree it's a good idea to put the default at `False`!